lib/src/screens/pin_code/pin_code_widget.dart

```diff
@@ -165,109 +160,112 @@ class PinCodeState<T extends PinCodeWidget> extends State<T> {
                 style: TextStyle(
                     fontSize: 14.0,
                     fontWeight: FontWeight.normal,
-                    color: Theme.of(context)
-                        .accentTextTheme!
-                        .bodyText2!
-                        .decorationColor!),
-              ))
-        ],
-        Spacer(flex: 1),
-        Flexible(
+                    color: Theme.of(context).accentTextTheme!.bodyText2!.decorationColor!),
+              ),
+            )
+          ],
+          Spacer(flex: 1),
+          Flexible(
             flex: 24,
-            child: Container(
+            child: Center(
+              child: Container(
+                height: MediaQuery.of(context).size.height * 0.6,
+                width: MediaQuery.of(context).size.height * 0.6,
```

Comment:
Constrained box would be better here too, cause this will affect the mobile view slightly